packages/contracts-bedrock/contracts/test/Hashing.t.sol

```diff
@@ -6,20 +6,33 @@ import { Types } from "../libraries/Types.sol";
 import { Hashing } from "../libraries/Hashing.sol";
 import { Encoding } from "../libraries/Encoding.sol";
 
-contract Hashing_Test is CommonTest {
+contract Hashing_hashDepositSource_Test is CommonTest {
     function setUp() external {
         _setUp();
     }
 
+    /**
+     * @notice Tests that hashDepositSource returns the correct hash in a simple case.
+     */
     function test_hashDepositSource_succeeds() external {
-        bytes32 sourceHash = Hashing.hashDepositSource(
-            0xd25df7858efc1778118fb133ac561b138845361626dfb976699c5287ed0f4959,
-            0x1
+        assertEq(
+            Hashing.hashDepositSource(
+                0xd25df7858efc1778118fb133ac561b138845361626dfb976699c5287ed0f4959,
+                0x1
+            ),
+            0xf923fb07134d7d287cb52c770cc619e17e82606c21a875c92f4c63b65280a5cc
         );
+    }
+}
 
-        assertEq(sourceHash, 0xf923fb07134d7d287cb52c770cc619e17e82606c21a875c92f4c63b65280a5cc);
+contract Hashing_hashCrossDomainMessage_Test is CommonTest {
+    function setUp() external {
+        _setUp();
     }
```

Comment:
Nit: it'd be nice if this boilerplate was moved into a contract which is a parent of the other contracts.

Or the `CommonTest._setUp()` could just become a public `setUp()` function.

Reply:
I feel like there was a reason for `CommonTest` to have the prefix underscored set up function, i had trouble getting it to compile awhile ago when i was originally working on it, but i agree that would be best